src/configureStore.js

```diff
@@ -90,12 +90,25 @@ const serializeTransform = createTransform(
   { whitelist: ['txControllerReducer'] }
 )
 
+const EthContractsTransform = createTransform(
```

Comment:
this doesn't seem to be doing any real transformation

Reply:
It only picks out the ethContracts from accountReducer.